lib/stack_master/cli.rb

```diff
@@ -7,7 +7,7 @@ class CLI
 
     def initialize(argv, stdin=STDIN, stdout=STDOUT, stderr=STDERR, kernel=Kernel)
       @argv, @stdin, @stdout, @stderr, @kernel = argv, stdin, stdout, stderr, kernel
-      Commander::Runner.instance_variable_set('@singleton', Commander::Runner.new(argv))
+      Commander::Runner.instance_variable_set('@instance', Commander::Runner.new(argv))
```

Comment:
Is there another way of achieving the same thing that doesn't set internals of commander?

Reply:
Not that I can see without further interactions with private `commander` internals, ie monkey patching.

I think this is a symptom of bad design in the `commander` gem and is another example of the Singleton Method pattern causing problems.